paper-progress.html

```diff
@@ -325,7 +329,10 @@
 
       this.secondaryProgress = secondaryProgress;
 
-      this.setAttribute('aria-valuenow', value);
+      if (!this.indeterminate) {
+        this.setAttribute('aria-valuenow', value);
+      }
+
       this.setAttribute('aria-valuemin', min);
       this.setAttribute('aria-valuemax', max);
```

Comment:
From the jQuery UI example at least, it seems that they don't set `aria-valuenow` and `aria-valuemax` when indeterminate, so that probably means they don't reset it properly if it changes. Min/max/value don't really make sense in the indeterminate case, so I rather not set them in that case.